### PR TITLE
[shopsys] nginx has now same limit for file size as is set in php.ini

### PR DESCRIPTION
**Pull request by TomasLudvik**
| Q             | A
| ------------- | ---
|Description, reason for the PR| Default limit of client_max_body_size that affects uploaded files in Nginx is 1MB. In php-ini-overrides.ini is limit set to 32MB. Default file size in administration is set to 2MB. We want to have services configured the same by default to 32MB and for appliacation configuration in administration forms we will keep limit of 2MB.
|New feature| No 
|[BC breaks](https://github.com/shopsys/shopsys/blob/master/docs/contributing/backward-compatibility-promise.md)| No 
|Fixes issues| ... 
|Standards and tests pass| Yes
|Have you read and signed our [License Agreement for contributions](https://www.shopsys.com/license-agreement)?| Yes

### CODE REVIEW

**Comment by vitek-rostislav** [comment]
Hi Tom, thanks for the PR, I will make sure it is processed in the current or the upcoming sprint (ie. within 2 weeks) :wink:

**Comment by boris-brtan** [comment]
rebased on master with fixing the conflicted UPGRADE notes

**Comment by boris-brtan** [comment]
for k8s can be tested by modifying 32m into 1m or so via command `kubectl edit -n tl-max-file-upload-nginx  ingress`

File: project-base/kubernetes/ingress.yml [hunk 1]
```diff
@@ -2,6 +2,8 @@ apiVersion: extensions/v1beta1
 kind: Ingress
 metadata:
     name: shopsys
+    annotations:
+        nginx.ingress.kubernetes.io/proxy-body-size: "32m"
```
Comment (grossmannmartin) [inline project-base/kubernetes/ingress.yml]: should it be like string?
in https://github.com/kubernetes/ingress-nginx/blob/master/docs/user-guide/nginx-configuration/annotations.md#custom-max-body-size they don't use quotes.

File: docs/upgrade/UPGRADE-unreleased.md [hunk 2]
```diff
@@ -74,6 +74,25 @@ There you can find links to upgrade notes for other versions too.
...
+        root /var/www/html/web;
+        server_tokens off;
+    +   client_max_body_size 32M;
+    ```
+    - update your [ingress.yml](../../project-base/kubernetes/ingress.yml) config file
+        ```diff
+        metadata:
+            name: shopsys
+        +   annotations:
+        +       nginx.ingress.kubernetes.io/proxy-body-size: "32m"
```
Comment (grossmannmartin) [inline docs/upgrade/UPGRADE-unreleased.md]:
```suggestion
        +       nginx.ingress.kubernetes.io/proxy-body-size: 32m
```

File: project-base/kubernetes/ingress.yml (hunk 1 above)
Comment (grossmannmartin) [inline project-base/kubernetes/ingress.yml]:
```suggestion
        nginx.ingress.kubernetes.io/proxy-body-size: 32m
```

**Pull request merged**